Exercise/compute_lowest_terms.py

```diff
@@ -1,40 +1,73 @@
 # TODO: complete the function "lowest_terms" below
 
 def lowest_terms(x):
+	"""
+	Reducing fractions basically requires finding factors and dividing by them.
+	Containers (lists) are created e.g cont, factors and results to hold these
+	generated values.
+	"""
 	x = x.split('/')
-	l = []
-	fac = []
-	res = []
+	cont = []
+	factors = []
+	result = []
+
+	# count identifies the number of negatives in the fraction to determine its sign
 	count = 0
+
+	# Separating negatives from values
 	for i in x:
 		if '-' in i:
 			x[x.index(i)] = i[1:]
 			count += 1
+
+	# Converting strings to integers
 	for j in x:
-		l.append(int(j))
-	copy = l[:]
+		cont.append(int(j))
```

Comment:
count = [int(val) for val in x]

shorter and more pythonic

Reply:
Sure, list comprehensions are easier, thank you